slm_lab/spec/benchmark/sac/sac_lunar.json

```diff
@@ -0,0 +1,61 @@
+{
+  "sac_lunar": {
+    "agent": [{
+      "name": "SoftActorCritic",
+      "algorithm": {
+        "name": "SoftActorCritic",
+        "action_pdtype": "RelaxedOneHotCategorical",
+        "action_policy": "default",
+        "gamma": 0.99,
+        "training_frequency": 1
+      },
+      "memory": {
+        "name": "Replay",
+        "batch_size": 256,
+        "max_size": 100000,
+        "use_cer": true
+      },
+      "net": {
+        "type": "MLPNet",
+        "hid_layers": [64, 64, 32],
+        "hid_layers_activation": "relu",
+        "init_fn": "orthogonal_",
+        "clip_grad_val": 0.5,
+        "loss_spec": {
+          "name": "MSELoss"
+        },
+        "optim_spec": {
+          "name": "Adam",
+          "lr": 0.01
```

Comment:
wrong lr?

Reply:
0.005 works better for LunarLander. solved within 200k frames